common/reviews/api/heft-configuration-loader.api.md

```diff
@@ -0,0 +1,68 @@
+## API Report File for "@rushstack/heft-configuration-loader"
```

Comment:
This name is pretty verbose, and sounds a bit like a Webpack loader plugin.  Should we shorten `@rushstack/heft-configuration-loader` to `@rushstack/heft-config`?